### PR TITLE
NXP-28804: add quickfilter for docs in retention

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

File: i18n/messages-fr.json [hunk 1]
```diff
@@ -1150,6 +1150,7 @@
   "ui.label.quickFilters.hasLegalHold": "Under legal hold",
   "ui.label.quickFilters.hasNotLegalHold": "Not under legal hold",
   "ui.label.quickFilters.withoutRule": "Sans règle jointe",
+  "ui.label.quickFilters.withRule" : "Avec une règle  jointe",
   "untrashDocumentsButton.confirm.untrashDocuments": "Restaurer le(s) document(s) sélectionné(s) ?",
```
Comment (RSalem07) [inline i18n/messages-fr.json:1154]: no space between the the key and `:` ;)

File: i18n/messages.json [hunk 2]
```diff
@@ -1150,6 +1150,7 @@
   "ui.label.quickFilters.hasLegalHold" : "Under legal hold",
   "ui.label.quickFilters.hasNotLegalHold" : "Not under legal hold",
   "ui.label.quickFilters.withoutRule" : "Without attached rule",
+  "ui.label.quickFilters.withRule" : "With attached rule",
   "untrashDocumentsButton.confirm.untrashDocuments": "Untrash selected document(s)?",
```
Comment (RSalem07) [inline i18n/messages.json:1154]: same as above (no space :))

File: i18n/messages-fr.json (hunk 1 above)
Comment (gcarlin) [reply i18n/messages-fr.json:1154]: fixed

File: i18n/messages.json (hunk 2 above)
Comment (gcarlin) [reply i18n/messages.json:1154]: fixed

**Comment by gcarlin** [comment]
That's correct! I've edited the result layout: so the quickfilter works, there were some logic on the bulk actions displayed on the document listing

**Comment by jaubenque** [comment]
I confirm this is just for search purpose, no action needed. Thanks

**Pull request merged**